docs/template/public/less/_homepage.less

```diff
@@ -0,0 +1,259 @@
+// Home page styles 
+
+@import '_variables';
+
+header.hyHomeHeaderMain {
+    background: url("/images/home/banner_bg.png") top center repeat-x ;
+    margin-top: -20px;
+    margin-left: -15px;
+    margin-right: -15px;
+    height: 440px;
+}
+
+header.hyHomeHeaderMinor {
+    background-color: @screen-bg_gd;
+    margin-left: -16px;
+    margin-right: -15px;
+}
+
+.hyHomePage {
+
+    h1 {
+        font-size: 35px;
+    }
+
+    h3 {
+        font-size: 40px;
+    }
+
+    .hyHomeHeader p, .hyHomeHeader a {
+        font-size: 18px;
+        font-family: 'Dosis';
+    }
+
+    .hyHomeHeader {
```

Comment:
it doesn't look like we need this rule at all, because text-weight doesn't even exist